config/locales/en.yml

```diff
@@ -36,6 +36,8 @@ en:
         create: "Add"
   activerecord:
     attributes:
+      gsa18f/procurement:
+        recurring: "Recurring charge"
```

Comment:
We haven't been trying at all...I was just looking for ways to take as much as possible out of the templates. The fact that it would make internationalization easier is a (distant) secondary benefit.